### PR TITLE
Bug/strapibuild

### PR DESCRIPTION
**Pull request by parulmishra**
Creating a new pull request to test artifactory changes

### CODE REVIEW

File: react-frontend/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # pull official base image
-FROM node:12.17.0-alpine
+FROM docker-remote.artifacts.developer.gov.bc.ca/node:lts-alpine
```
Comment (thegentlemanphysicist) [inline react-frontend/Dockerfile:2]: Do these still build locally ?

File: strapi-app/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # pull official base image
-FROM node:12.17.0-alpine
+FROM docker-remote.artifacts.developer.gov.bc.ca/node:lts-alpine
```
Comment (thegentlemanphysicist) [inline strapi-app/Dockerfile:2]: Do these still build locally?

**Comment by thegentlemanphysicist** [comment]
This PR is probably no longer necessary now that strapi builds are using the artifactory creds

**Comment by parulmishra** [comment]
This issue is already resolved hence closing the PR

**Pull request closed without merging**